### DISCUSSION TITLE
Installing IIAB on my laptop [Ubuntu 22.04]

### DISCUSSION THREAD

**Discussion by babalegba** [iiab/iiab#3684] [Q&A] [answered]
Please I need help on how to install IIAB on my laptop. The OS is ubuntu 22.04

**Comment by holta** [comment, marked as answer]
Please read the instructions at http://FAQ.IIAB.IO = https://wiki.iiab.io/go/FAQ

- In particular, the 1-line command to begin installing Internet-in-a-Box software is shown here:

   https://download.iiab.io

- Also see the Wi-Fi hotspot tips here:

   "19 Can I create a Wi-Fi hotspot using an old laptop?"

   https://wiki.iiab.io/go/FAQ#Can_I_create_a_Wi-Fi_hotspot_using_an_old_laptop%3F

**Reply by babalegba** [reply]
Thanks

While running sudo iiab, I am getting the attached error

![error](https://github.com/iiab/iiab/assets/120962387/c2508c39-3515-4388-976f-790a38054163)

![error](https://github.com/iiab/iiab/assets/120962387/1160e6a9-706c-4dd5-8621-e9574bc67cd4)

**Comment by tim-moody** [comment]
somehow you have an extra line with only a 0 in it in /etc/iiab/local_vars.yml. use nano or some editor to simply remove that line and rerun.

**Reply by babalegba** [reply]
I have corrected the error but I ran into another

**Comment by tim-moody** [comment]
How recent is this install? I notice that python is 3.10. Some time ago there was an issue with local_vars.yml getting corrupted, but it has not been seen with the latest version.

**Reply by babalegba** [reply]
The installation was done today

**Comment by babalegba** [comment]
I ran into another error

![err1](https://github.com/iiab/iiab/assets/120962387/624b37bb-26f7-42dc-8ed2-a6d1d9a6b29d)
![err2](https://github.com/iiab/iiab/assets/120962387/bb2af0a1-caf5-4016-a415-a4a5deca0a02)
![err3](https://github.com/iiab/iiab/assets/120962387/30cb8df1-2f91-4705-a7f1-8d7a0a4c3f64)

**Comment by holta** [comment]
1) @babalegba what happens if you reboot and try `sudo iiab` again?

   Does the remoteit error happen every single time you run `sudo iiab` ?

2) If absolutely necessary, change Line 141 of /etc/iiab/local_vars.yml from `remoteit_install: True` to...

    `remoteit_install: False`

    ...then run `sudo iiab` again.

**Reply by babalegba** [reply]
Many thanks to everyone. I was finally able to install it after changing remoteit_install to False

**Reply by assuredclean** [reply]
Same problem here and remoteit_install: False fixed it!

**Reply by jvonau** [reply]
Think TPTB should change the default until somebody fixes remoteIT

**Reply by holta** [reply]
RemoteIT works most of the time.

What exactly is intermittent is not yet understood.

[ Possibly related... ]

- #3705
